api/libs/scenarios-planning-unit/src/domain/lock-planning-units.ts

```diff
@@ -0,0 +1,11 @@
+import { differenceWith } from 'lodash';
+
+export const lockPlanningUnits = (
+  planningUnitsIds: string[],
+  availablePlanningUnitsIds: string[],
+): { errors: string[] } => {
+  const diff = differenceWith(planningUnitsIds, availablePlanningUnitsIds);
+  return {
+    errors: diff.map((missingId) => `Missing ${missingId}`),
```

Comment:
IIRC we tried to improve these error messages recently, while trying to not make them too verbose as they actually only differ by the PU id... Maybe just `Planning Unit with id ${missingId} is not part of this project`?

Reply:
or tweak this into a `canPlanningUnitsBeLocked()` and return true if diff is empty, false otherwise?

Reply:
I would stay with returning message (or maybe ids in future) and let consumer decide.

Reply:
@hotzevzl fixed in https://github.com/Vizzuality/marxan-cloud/pull/224/commits/8b9c4a7bbed5d99bc818bedd331199dc29415f8e (message)